### ISSUE TITLE
Email with new jobs that match criteria?

### ISSUE THREAD

**Issue by Keyser5023** [solve64/userscripts#1]
This is Sam R from wyzant; you mentioned the possibility of a recurring task that would send an email with jobs that meet certain criteria. Is it possible to have an email sent only if there are new jobs available? That would be awesome, but otherwise I think it would just clog my inbox. Thank you again!!

**Comment by solve64** [comment]
Yes, it can send the email only if there are new jobs available that match your criteria. For the job to run, you would need to have the program running on your computer that would always need to be on and connected to the internet for it to work. Its schedule can be configured so it's run every 30 minutes, every 60 minutes, etc. For the filtering, I can add whatever filters you want such as minimum recommended/required rate, required keywords, ignored keywords, etc.

Are you still interested given that?

**Comment by Keyser5023** [comment]
Yes, that would be really useful, so I wouldn't have to manually check the job postings constantly. What do you think would be a fair payment for you to put that together for me? Thanks!

**Comment by solve64** [comment]
Sorry for the back and forth, but I found a way to incorporate the Notification into my existing userscript and confirmed it works. Doing it this way will make it so you don't have to enter your Wyzant or email credentials into a program that some random guy on the internet (i.e. me) made -- you'll just have to install the updated userscript (like you did the first time) and leave the Wyzant jobs page open in the background. I tested it with a simple notification that waits 10 seconds then sends the notification. I opened the jobs page, switched to another tab, and even minimized the browser window and I still got a notification after 10 seconds on my system (Linux+Firefox) but should also work for Chrome/etc on mobile, Windows, Mac, etc.

At this point to finish it, I'd need to update the userscript to poll the jobs URL every x minutes and if it sees a newly added job since you previously loaded the page, then it will send the notification and when you click it, it will refresh the page to show the new job(s) at the top of the list.

For payment, $200 works for me as a one-time Ko-Fi payment. Does that work for you? Don't send the payment yet. I'd want you to make sure the simple notification test I made works on your system(s) before I finish the work then share the completed userscript to you. Do you want it shared to you alone, or update the existing userscript here that's publicly available. I don't mind either way.

**Comment by Keyser5023** [comment]
$200 is totally fine, but I'd prefer if you not make it publicly available (feel free to sell it to other people for a similar price, though). Where can I get the userscript to test the notification on my system?

**Comment by solve64** [comment]
Ok, sounds good. Send me an email to solver@solve64.com then I'll reply from my personal email with the notification test attached. Thanks
